generator/main.ts

```diff
@@ -152,7 +152,9 @@ async function main() {
 
     const routeDir = path.join("public", slug)
 
-    fs.mkdirSync(routeDir)
+    if (!fs.existsSync(routeDir)){
+      fs.mkdirSync(routeDir)
+    }
```

Comment:
This supports adding images to a post in a `/static/<slug>` dir.

Reply:
I'll update `content/build.md` w/ that info.